packages/nextjs/app/my-grants/page.tsx

```diff
@@ -51,32 +52,41 @@ const MyGrants: NextPage = () => {
             </div>
           )}
           <p className="m-0">{grant.description}</p>
-          <p className={`badge ${badgeBgColor[grant.status]}`}>{grant.status}</p>
-          {grant.approvedTx && (
-            <a
-              href={getBlockExplorerTxLink(Number(grant.txChainId), grant.approvedTx)}
-              target="_blank"
-              rel="noopener noreferrer"
-              className="ml-4 underline underline-offset-4 text-xs"
-            >
-              50% approve tx <ArrowTopRightOnSquareIcon className="h-4 w-4 inline" />
-            </a>
-          )}
-          {grant.completedTx && (
-            <a
-              href={getBlockExplorerTxLink(Number(grant.txChainId), grant.completedTx)}
-              target="_blank"
-              rel="noopener noreferrer"
-              className="ml-4 underline underline-offset-4 text-xs"
-            >
-              50% complete tx <ArrowTopRightOnSquareIcon className="h-4 w-4 inline" />
-            </a>
-          )}
-          {grant.status === PROPOSAL_STATUS.APPROVED && (
-            <button onClick={() => openModal(grant)} className="btn btn-primary float-right">
-              Submit build
-            </button>
-          )}
+          <div className="flex items-center justify-between">
+            <div className="flex items-center">
+              <p className={`badge ${badgeBgColor[grant.status]}`}>{grant.status}</p>
+              {grant.note && grant.note.trim().length > 0 && (
+                <div className="inline-block ml-1 tooltip tooltip-bottom cursor-pointer" data-tip={grant.note}>
```

Comment:
Super nitpick, ignore if you want 🙌

(?) Tooltip icon seems not aligned with badge. Should we remove the `inline-block` / `inline`?